docs/user-guide/faq.md

```diff
@@ -87,6 +87,54 @@ You can also take advantage of the `selector-*` rules to ban certain types of se
 
 If you're using SUITCSS, you might want to use [their shareable config](https://github.com/suitcss/stylelint-config-suitcss).
 
+## How do I disallow single-line blocks?
+
+```css
+  a { color: red; }
+/** ↑                      ↑
+ * Declaration blocks like this */
+```
+
+Use the `block-opening-brace-newline-after` and `block-opening-brace-newline-before` rules together. For example, this config:
+
+```json
+{
+  "block-opening-brace-newline-after": ["always"],
+  "block-closing-brace-newline-before": ["always"]
+}
+```
+
+Would allow:
+
+```css
+a {
+  color: red;
+}
+```
+
+But not this:
+
+```css
+a { color: red;
+}
+```
+
+This:
+
+```css
+a {
+color: red; }
+```
+
+Or this:
+
+```css
+a { color: red; }
+```
+
+To allow single line blocks, use the `"always-multi-line"` option for both rules.
```

Comment:
How about: To allow single-line blocks but enforce newlines with multi-line blocks, ...